src/main/java/de/berlin/lostberlin/service/mail/Params.java

```diff
@@ -3,34 +3,34 @@
 import java.util.Map;
 
 public class Params {
-	
-	private String toMail;
```

Comment:
So, what is finally here as padding? Tabs or spaces?

Reply:
you mean formatting, right? I auto-formatted all code, so it should be 4 spaces. I hope I understood you correctly.